src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -755,12 +755,12 @@ private DateTime GetDateTime(string key)
             {
                 if (ex is FormatException || ex is ArgumentException || ex is InvalidCastException)
                 {
-                    throw LogHelper.LogExceptionMessage(new SecurityTokenException(LogHelper.FormatInvariant(LogMessages.IDX12700, key, (dateValue ?? "<null>")), ex));
+                    throw LogHelper.LogExceptionMessage(new SecurityTokenException(LogHelper.FormatInvariant(LogMessages.IDX12700, key, LogHelper.MarkAsNonPII((dateValue ?? "<null>"))), ex));
```

Comment:
nit: not your change but seems odd the null string format is different here than elsewhere

Reply:
Updated to "Null" for consistency